Solutions/GreyNoiseThreatIntelligence/Data Connectors/azuredeploy_Connector_GreyNoiseAPISentinel_AzureFunction.json

```diff
@@ -0,0 +1,238 @@
+              ],
+              "kind": "functionapp,linux",
+              "identity": {
+                  "type": "SystemAssigned"
+              },
+              "properties": {
+                  "name": "[variables('FunctionName')]",
+                  "serverFarmId": "[resourceId('Microsoft.Web/serverfarms', variables('FunctionName'))]",
+                  "httpsOnly": true,
+                  "clientAffinityEnabled": true,
+                  "alwaysOn": true,
+                  "reserved": true,
+                  "siteConfig": {
+                    "linuxFxVersion": "Python|3.10"
+                  }
+              },
+              "resources": [
+                  {
+                      "apiVersion": "2018-11-01",
+                      "type": "config",
+                      "name": "appsettings",
+                      "dependsOn": [
+                          "[concat('Microsoft.Web/sites/', variables('FunctionName'))]"
+                      ],
+                      "properties": {
+                          "FUNCTIONS_EXTENSION_VERSION": "~4",
+                          "FUNCTIONS_WORKER_RUNTIME": "python",
+                          "APPINSIGHTS_INSTRUMENTATIONKEY": "[reference(resourceId('Microsoft.insights/components', variables('FunctionName')), '2015-05-01').InstrumentationKey]",
+                          "APPLICATIONINSIGHTS_CONNECTION_STRING": "[reference(resourceId('microsoft.insights/components', variables('FunctionName')), '2015-05-01').ConnectionString]",
+                          "AzureWebJobsStorage": "[concat('DefaultEndpointsProtocol=https;AccountName=', toLower(variables('FunctionName')),';AccountKey=',listKeys(resourceId('Microsoft.Storage/storageAccounts', toLower(variables('FunctionName'))), '2019-06-01').keys[0].value, ';EndpointSuffix=',toLower(variables('StorageSuffix')))]",
+                          "WEBSITE_CONTENTAZUREFILECONNECTIONSTRING": "[concat('DefaultEndpointsProtocol=https;AccountName=', toLower(variables('FunctionName')),';AccountKey=', listKeys(resourceId('Microsoft.Storage/storageAccounts', toLower(variables('FunctionName'))), '2019-06-01').keys[0].value, ';EndpointSuffix=',toLower(variables('StorageSuffix')))]",
+                          "WEBSITE_CONTENTSHARE": "[toLower(variables('FunctionName'))]",
+                          "WORKSPACE_ID": "[parameters('WORKSPACE_ID')]",
+                          "GREYNOISE_KEY": "[parameters('GREYNOISE_KEY')]",
+                          "TENANT_ID": "[parameters('TENANT_ID')]",
+                          "CLIENT_ID": "[parameters('CLIENT_ID')]",
+                          "CLIENT_SECRET": "[parameters('CLIENT_SECRET')]",
+                          "GREYNOISE_CLASSIFICATIONS": "[parameters('GREYNOISE_CLASSIFICATIONS')]",
+                          "WEBSITE_RUN_FROM_PACKAGE": "https://github.com/Azure/Azure-Sentinel/raw/db458a54839b084eac0e70bbe6e2a41f34f37e2b/Solutions/GreyNoiseThreatIntelligence/Data%20Connectors/GreyNoiseAPISentinelConn.zip"
```

Comment:
Zip package download link should have the master branch, currently its on your branch.

Reply:
How do I get it on Master Branch? Do I need a PR with just the zip file?

Reply:
see https://github.com/Azure/Azure-Sentinel/pull/9010

Reply:
Just replace the branch name with "master"

Reply:
Regarding this: I'm following the instructions here: https://github.com/Azure/Azure-Sentinel/blob/master/DataConnectors/Templates/Connector_REST_API_AzureFunctionApp_template/README.md#3-create-an-encoded-url which say to get my zip file up into master so I can update the link in azure_deploy file.